src/modules/addReservation.js

```diff
@@ -0,0 +1,82 @@
+let tempID;
+
+const renderReservations = async () => {
+  try {
+    const baseURL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+    const appID = 'fvEG8bcfusuKIAC9Au4g';
```

Comment:
- [ ] Kindly avoid code repetition here as well, these constants had already been declared in `addComment.js` file; implementing this will make your codebase cleaner. 👍